webpack.config.js

```diff
@@ -6,9 +6,12 @@ module.exports = {
   },
   devtool: 'source-map',
   module: {
+    preloaders: [
+      { test: /\.js$/, loader: 'jscs-loader', exclude: /\/(spec|node_modules)/ }
```

Comment:
I think it's better to have the -loader part because it's the loader that is dependent on the 'jscs' package.